crates/libs/metadata/Cargo.toml

```diff
@@ -12,7 +12,8 @@ default-target = "x86_64-pc-windows-msvc"
 targets = []
 
 [dependencies.windows-sys]
-version = "0.36.1"
+path = "../sys"
+version = "0.38"
```

Comment:
@kennykerr This will need your 👀. Probably can't be published like this?

Reply:
I think it has to be changed back before being published - the `metadata` crate, that is. That's what happened in the `0.38` release as well.

I've previously (in https://github.com/microsoft/windows-rs/commit/4c7ae6bdc0338e2aee9b578b64306b4696816acf#r76875081) suggested to simply _not_ bump the version of `windows-sys` if it hadn't seen a release anyway. That'd solve this and more issues.
(Afaik `cargo` downloads a published crate from `crates.io` for every dependency before building and publishing the current package, instead of working on local source)